statute.md

```diff
@@ -22,6 +22,10 @@ Berlin TWC has as its primary goal the empowerment of Tech Workers in their indu
 The application and modification of this document must be driven by the principles of fairness, horizontality, inclusivity organizational scalability and safety of the members. 
 
 
+1.3.1
+
+The members of Berlin TWC should be considered equal among each other and equal to every other worker. A member is expected to show solidarity to other workers through the pursuit of the organization's goals. Each member contributes at the best of their intellectual, physical and economic capabilities.  
```

Comment:
I would change last line to be: `Every member is expected to contribute to best of their knowledge and strength to the best of their ability`

Reply:
I would also propose a definition: "A member is of good standing, if they have attended 3 events within the past 6 months". This would mitigate a situate where someone was involved/active say from June-December 2019 and then shows up in April 2020, without having attended anything in between.

Reply:
can you please review the sentence you proposed? I guess something went wrong with the copy-pasting?

Reply:
It will be changed into: "Every member is expected to contribute to best of their knowledge and strength"